### ISSUE TITLE
wrong version of holidays/countries/united_arab_emirates.py in the release

### ISSUE THREAD

**Issue by niradar** [vacanza/holidays#2054]
## Bug Report
holidays/countries/united_arab_emirates.py in the release is different than the version in github.

In github:
-------------
    def _populate_public_holidays(self):
        # The resting days are Saturday and Sunday since Jan 1, 2022.
        # https://time.com/6126260/uae-working-days-weekend/
        self.weekend = {FRI, SAT} if self._year <= 2021 else {SAT, SUN}
------------

And in the release this line is missing and causing a bug

### Expected Behavior

Include the updated file from the github in the release

### Actual Behavior

-------------
    print(holidays.AE.weekend)
-------------

returns {4, 5} but needs to be {5, 6}

### Steps to Reproduce the Problem
-------------
    import holidays
    print(holidays.AE.weekend)
-------------

### Environment

- OS: Windows and Linux
- Python version: 3.12
- holidays version: 0.58 using pip install

**Comment by PPsyrius** [comment]
This is because the new changes on the `dev` branch was just merged in a few days ago (see PR #2048) and would only be included from `v0.59` release onwards.

**Comment by arkid15r** [comment]
> This is because the new changes on the `dev` branch was just merged in a few days ago (see PR #2048) and would only be included from `v0.59` release onwards.

Right, @niradar it seems you need to check the code against the [main branch](https://github.com/vacanza/holidays/tree/main).
